src/sql/parts/modelComponents/queryTextEditor.ts

```diff
@@ -129,13 +134,28 @@ export class QueryTextEditor extends BaseTextEditor {
 		// number of lines that wrap). Finally, viewportColumn is calculated on editor resizing automatically; we can use it to ensure
 		// that the viewportColumn will always be greater than any character's column in an editor.
 		let numberWrappedLines = 0;
-		for (let line = 1; line <= lineCount; line++) {
-			if (editorWidgetModel.getLineMaxColumn(line) >= this._config.editor.layoutInfo.viewportColumn - 1) {
-				numberWrappedLines += Math.ceil(editorWidgetModel.getLineMaxColumn(line) / this._config.editor.layoutInfo.viewportColumn);
+		let shouldAddHorizontalScrollbarHeight = false;
+		if (!this._editorWorkspaceConfig) {
```

Comment:
You can add a config changed listener so this gets refreshed. I added in a recent review. That'll ensure we always have the right value, and on changing word wrap on/off we can then re-run this function to update size.